docs/pages/includes/database-access/create-user.mdx

```diff
@@ -36,7 +36,7 @@ $ tctl users add \
 | `--db-names` | List of logical databases (aka schemas) the user will be allowed to connect to within a database server. A wildcard allows any database. |
 
 <Admonition type="warning">
-  Database names are only enforced for PostgreSQL and MongoDB databases.
+  Database names are only enforced for PostgreSQL, MongoDB, and Cloud Spanner databases.
```

Comment:
What is the official product name? Is it _GCP Cloud Spanner_, _Cloud Spanner_, or just _Spanner_?


https://cloud.google.com/spanner suggests the latter.

Reply:
Google inconsistently refers to it as either Cloud Spanner or just "Spanner".
I have seen "Cloud Spanner" more than Spanner though.

https://cloud.google.com/spanner/docs/reference/rpc
https://status.cloud.google.com/products/EcNGGUgBtBLrtm4mWvqC/history